src/components/LinkList/_link-list.scss

```diff
@@ -0,0 +1,39 @@
+// ----------------------------------
+// UI Components
+// Media (in-page Image and Video)
+// ----------------------------------
+// Design System 1.0 Alpha
+// 2019-11-11
+// ----------------------------------
+
+.cc-link-list {
+  @extend %list-clean;
```

Comment:
I have a question: what does `cc-` mean? is it `corporate-component`?

Reply:
👍 correct